lib/transbank/sdk/patpass/errors/transaction_commit_error.rb

```diff
@@ -0,0 +1,8 @@
+module Transbank
+  module Patpass
+    module Errors
+      class TransactionCommitError < PatPassError
```

Comment:
This should be `PatpassError` instead of `PatPassError`

Reply:
Yes! It existed, but I've changed it and update the PR